bin/skipped_tests.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Please add a comment explaining what this file does.

Reply:
got it!